usgs-details.html

```diff
@@ -170,6 +170,17 @@
             };
         }
 
+        // Handle XHR errors by using fallback data
+        function handleXHRError(metric) {
+            return function() {
+                var value = metric.fallback[new Date().getMonth()],
+                    units = metric.units,
+                    isFallback = true;
+
+                return renderData(value, units, metric, isFallback);
```

Comment:
Consider adding a `console.warn` statement here as done in `handleXHRResponse` for the error case.